Source/WebKit/UIProcess/API/ios/WKWebViewIOS.mm

```diff
@@ -1660,6 +1660,16 @@ - (BOOL)_isWindowResizingEnabled
 
 #endif // HAVE(UIKIT_RESIZABLE_WINDOWS)
 
+#if ENABLE(INTERACTION_REGIONS_IN_EVENT_REGION)
+- (void)_setIsLinkPreview:(BOOL)isLinkPreview
```

Comment:
`_setIsLinkPreview` could be renamed now to `_setEnableBuildingInteractionRegions`

Reply:
Yes, I was expecting the SPI to get renamed too.